### DISCUSSION TITLE
Model Outline Color

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1952] [Support Q&A] [answered]
I am trying to draw outline of a 3D model. When I load the model on OpenSceneGraph there is no problem with color of the outline as shown on below picture.

![image](https://user-images.githubusercontent.com/5169314/151729942-cbd10916-9230-445c-bcd0-1874dcc385f2.png)

Bu when I try to do the same thing with osgEarth, the outline does not show the expected color. It shows always black outline. What could be the reason?

![image](https://user-images.githubusercontent.com/5169314/151730599-2590996b-4fd6-4ff8-83ea-a2385a4bae40.png)

**Comment by nebsar** [comment, marked as answer]
Although I am very new to shaders, I created a modified osgFx::Outline using shaders and now I can change the color of the outline as I want.

![image](https://user-images.githubusercontent.com/5169314/153365586-9fa40b38-4ca5-49e1-9fb7-e7b6f2392e95.png)

**Reply by csuxing** [reply]
I also encountered the same problem, how did you achieve it? Thank you bro.
